libs/user/src/assets/i18n/fr.json

```
@@ -5,14 +5,17 @@
 	"generic.or": "Ou",
 	"login.access_request": "souhaite accéder à Nuclia en votre nom et être en mesure de :",
 	"login.boarding_title": "Embarquer avec Nuclia",
-	"login.check_email": "Vérifiez vos emails!",
-	"login.check_email_details": "Cliquez sur le lien que vous venez de recevoir à <strong>{{ email }}</strong> pour finaliser votre compte. Si vous ne voyez rien... Vérifiez votre dossier spam 🙃 !",
+	"login.check_email.check_spams": "ou vérifiez votre dossier spam.",
```

usually that's more:
```suggestion
	"login.check_email.check_spams": "ou vérifiez vos courriers indésirables.",
```